pkg/usersignup/usersignup_test.go

```diff
@@ -31,6 +31,12 @@ func TestTransformUsername(t *testing.T) {
 	assertName(t, "crt-me-crt", "_me_")
 	assertName(t, "crt-me-crt", "-me-")
 	assertName(t, "crt-12345", "12345")
+	assertName(t, "thisisabout20charact", "thisisabout20characters@email.com")
+	assertName(t, "isexactly20character", "isexactly20character@email.com")
+	assertName(t, "isexactly20character", "isexactly20character")
+	assertName(t, "isexactly21characte", "isexactly21characte-r") // shortened 20 character will end in hyphen
+	assertName(t, "isexactly20charactr", "isexactly20charactr-")  // but ending in hyphen
+	assertName(t, "thisis19characters-c", "thisis19characters-")  // suffix -crt is added before truncating string
 }
 
 var dnsRegExp = "^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"
```

Comment:
I guess we can change this according to @mfrancisc comment too?

Reply:
done in [9a9de94](https://github.com/codeready-toolchain/toolchain-common/pull/309/commits/9a9de94456dbb022f7aa75a6dec18f0689fe2171)